frontend/lib/features/audio-player/audio-player-screen.widget.dart

```diff
@@ -0,0 +1,207 @@
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 20.0),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children:[
+                            Text(widget.title, style: StandardText.headline6,),
+                            const SizedBox(height: 5,),
+                            Text(widget.speaker, style: StandardText.subtitle2,),
+                          ],
+                        ),
+                        const Icon(Icons.favorite_border_outlined) // Nur zum Testzweck
+                      ],
+                    ),
+                  ),
+                  const SizedBox(height: 19.0,),
+                  CustomProgressBar(positionDataStream: _positionDataStream, player: _player),
+                  const SizedBox(height: 44.0,),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      IconButton(
+                        onPressed: () {
+                          if(_player.position < const Duration(seconds: 10)){
+                            _player.seek(Duration.zero);
+                          } else {
+                            _player.seek(_player.position - const Duration(seconds: 10));
+                          } 
+                        },
+                        icon: const Icon(Icons.replay_10_rounded),
+                        iconSize: 60,
+                      ),
+                      const SizedBox(width: 27,),
+                      StreamBuilder<PlayerState>(
+                        stream: _player.playerStateStream,
+                        builder: (context, snapshot) {
+                          final playerState = snapshot.data;
+                          return _playerButton(playerState);
+                        }
+                      ),
+                      const SizedBox(width: 27,),
+                      IconButton(
+                        onPressed: () {
+                         if(_player.position > _player.duration! - const Duration(seconds: 10)){
+                            _player.seek(_player.duration);
+                          } else {
+                            _player.seek(_player.position + const Duration(seconds: 10));
+                          } 
+                        },
+                        icon: const Icon(Icons.forward_10_rounded),
+                        iconSize: 60,
+                      ),
+                    ],
+                  ),
+                  const SizedBox(height: 80,)
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+Widget _playerButton(PlayerState? playerState) {
```

Comment:
move widget to it's own file. only one widget per file. create a meaningful directory structure